.github/workflows/lint.yml

```diff
@@ -0,0 +1,62 @@
+# Run linters automatically on pull requests.
+name: Lint sros2 repo
+on:
+  pull_request:
+  push:
+
+jobs:
+  ament_copyright:
+    name: ament_copyright
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-lint@0.0.5
+      with:
+        linter: copyright
+        package-name: |
+          sros2
+          sros2_cmake
+
+  ament_lint_cmake:
+    name: ament_lint_cmake
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-lint@0.0.5
+      with:
+        linter: copyright
+        package-name: |
+          sros2_cmake
```

Comment:
Aren't you linting `sros2_cmake` in the above job?

Reply:
indeed, a copy-n-paste mistake, it should be `lint_cmake`, let me see if I can find a way to not have to list the linter name twice for every job.

Edit: actually the current lint action doesn't support linters with underscore/dashes so I guess we'll just remove this one for now

Reply:
As this is YAML, you should be able to use anchors and aliases.

But, [this](https://github.community/t5/GitHub-Actions/Support-for-YAML-anchors/td-p/30336) seems to suggest it's not supported in GH actions.

Reply:
Yeah hopefully it'll happen in the near future, other systems like travis and Circle do support it and it is pretty convenient.

Ended up using some matrix env hackery instead... https://github.com/ros2/sros2/pull/178/files#diff-e9f950f17198d3d5e3122a44230a09b9R31
not great but allows some deduplication.